areas.html

```diff
@@ -3,11 +3,23 @@
 <head>
     <link href="css/grid.css" rel="stylesheet">
     <style>
+        body {
+            margin: 0;
+        }
         .grid {
-            grid-template-areas: "a b b c"
-                                 "d b b c"
-                                 "e e e e";
-            width: 150px;
+            grid-template-columns: repeat(3, 1fr) repeat(2, 2fr);
+            grid-template-rows: repeat(5, 1fr);
+            grid-template-areas: "a b b . ."
+                                 ". b b . c"
+                                 "d b b . c"
+                                 "d . . . c"
+                                 "e e e e e";
+            color: #f00;
```

Comment:
I'd use a different color than `red` as people can get confused about this being a mistake or not.
I think letting it black would be fine.